build.gradle

```diff
@@ -167,8 +166,6 @@ sonarqube {
 	}
 }
```

Comment:
If jacoco is used for code coverage in profile sync then this shud be added -

 -project.tasks['sonarqube'].dependsOn jacocoTestReport